resources/l10n/da/LC_MESSAGES/loot.po

```diff
@@ -4,1267 +4,2086 @@
+
+#: src/gui/qt/plugin_item_model.cpp:179
+msgid "Index"
+msgstr "Indeks"
+
+#
+# File: src/gui/html/js/translateStaticText.js, line: 438
+# File: src/gui/html/js/translateStaticText.js, line: 438
+#: src/gui/qt/plugin_item_model.cpp:181
+msgid "Plugin Name"
+msgstr "Plugin navn"
+
+#
+# File: src/gui/html/js/translateStaticText.js, line: 384
+# File: src/gui/html/js/translateStaticText.js, line: 384
+#: src/gui/qt/search_dialog.cpp:99
+msgid "Search Cards"
+msgstr "Søg kort"
+
+#
+# File: src/gui/html/js/translateStaticText.js, line: 421
+# File: src/gui/html/js/translateStaticText.js, line: 421
+#: src/gui/qt/search_dialog.cpp:101
+msgid "Search cards"
+msgstr "Søg kort"
+
+#: src/gui/qt/search_dialog.cpp:105
+msgid ""
+"If checked, interprets the search text as a Perl-like regular expression."
+msgstr "Hvis markeret, bliver søgeteksten fortolket som et regulært udtryk."
+
+#: src/gui/qt/search_dialog.cpp:108
+msgid "Find Previous"
+msgstr "Find sidste"
+
+#: src/gui/qt/search_dialog.cpp:109
+msgid "Find Next"
+msgstr "Find næste"
+
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:148
+msgid "Groups Editor"
+msgstr "Gruppebehandler"
 
 #
 # File: src/gui/html/js/translateStaticText.js, line: 66
 # File: src/gui/html/js/translateStaticText.js, line: 214
 # File: src/gui/html/js/translateStaticText.js, line: 66
 # File: src/gui/html/js/translateStaticText.js, line: 214
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:150
+msgid "Group name"
+msgstr "Gruppe navn"
+
+#
+# File: src/gui/html/js/translateStaticText.js, line: 360
+# File: src/gui/html/js/translateStaticText.js, line: 605
+# File: src/gui/html/js/translateStaticText.js, line: 360
+# File: src/gui/html/js/translateStaticText.js, line: 605
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:151
+msgid "Add a new group"
+msgstr "Tilføj ny gruppe"
+
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:152
+msgid "Auto arrange groups"
+msgstr "Arranger grupper automatisk"
+
+#
+# File: src/gui/html/js/events.js, line: 396
+# File: src/gui/html/js/events.js, line: 396
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:169
+msgid "You have unsaved changes. Are you sure you want to discard them?"
+msgstr "Du har ikke-gemte ændringer. Er du sikker på, at du vil kassere dem?"
+
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:227
+msgid "No plugins are in this group."
+msgstr "Ingen plugins i denne gruppe."
+
+#
+# File: src/gui/html/js/translateStaticText.js, line: 438
+# File: src/gui/html/js/translateStaticText.js, line: 438
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:239
+#, c-format
+msgid "Plugins in %s"
+msgstr "Plugins i %s"
+
+#: src/gui/qt/groups_editor/groups_editor_dialog.cpp:260
+msgid "Group already exists!"
+msgstr "Gruppe eksistere allerede!"
+
+#
+# File: src/gui/state/game.cpp, line: 457
+# File: src/gui/state/game.cpp, line: 459
+#: src/gui/qt/plugin_editor/group_tab.cpp:56
+msgid "%1 (Group does not exist)"
+msgstr "Gruppen “%1%” findes ikke."
```

Comment:
The placeholders should match, so the translation should use `%1` in this string. Also, the context for this string is that it shows up in the **Group:** drop-down menu in the metadata editor when a plugin is assigned a group that is not defined in either the masterlist or userlist. It'd therefore be preferable to keep the same format as the original.

Reply:
Should be fixed in latest commit.